components/meta-packages/X-incorporation/Makefile

```diff
@@ -0,0 +1,26 @@
+#
```

Comment:
Please do not pollute this PR by incorporating other components than sunray-essential.  Thank you.